### DISCUSSION TITLE
maybe a setup.exe can better  build `GacLib` env

### DISCUSSION THREAD

**Discussion by Alafun** [vczh-libraries/Release#12] [Suggestions]
Maybe a setup.exe can better  build `GacLib` env.

**Comment by pzhlkj6612** [comment]
Maybe a [native NuGet package](https://docs.microsoft.com/en-us/nuget/guides/native-packages) is better...?

**Comment by vczh** [comment]
I don't quite follow you, do you suggest that I could release GacUI by pushing it to NuGet? What is setup.exe supposed to do?
